### ISSUE TITLE
Pagination support

### ISSUE THREAD

**Issue by danielloader** [marcusolsson/grafana-json-datasource#184]
I know this is a difficult ask as every API has a different implementation of pagination, and I'm dealing with a particularly non standard implementation [here ](https://docs.pingdom.com/api/?_ga=2.230003480.509660209.1590495493-1793431897.1589990976#tag/Results/paths/~1results~1{checkid}/get)

The issue being I need to keep incrementing the `offset` value with the `to` and `from` parameters provided by Grafana, until the offset yields zero results (because its outside the time range). 

Understand it's a loose brief here so ideas are welcome (or marking it as not feasible). 

Thanks!

**Comment by danielloader** [comment]
Definitely, I was hesitant to even open the ticket given every API does it differently, maybe some more input - even if it doesn't help me, could help others with this.

**Comment by userlerueda** [comment]
What about working with the "next" key in the body ? It could be a starting point.
